src/services/member/plugins/action/test/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTriggers, DiscriminatedItem } from '@graasp/sdk';
+
+import { AppDataSource } from '../../../../../plugins/datasource';
+import { Action } from '../../../../action/entities/action';
+import { saveActions } from '../../../../action/test/fixtures/actions';
+import { ItemTestUtils } from '../../../../item/test/fixtures/items';
+import { saveMember } from '../../../test/fixtures/members';
+
+const testUtils = new ItemTestUtils();
+const rawRepository = AppDataSource.getRepository(Action);
+
+export const getDateBeforeOrAfterNow = (dateDiff) => {
+  const date = new Date(); // Today's date
+  date.setDate(date.getDate() + dateDiff);
+  return date.toISOString();
+};
+
+export const saveActionsWithItems = async (member, { saveActionForNotOwnedItem = false } = {}) => {
+  const item = await testUtils.saveItem({ actor: member });
+  const actions = [
+    {
+      member,
+      createdAt: new Date().toISOString(),
+      type: ActionTriggers.Update,
+      item: item as unknown as DiscriminatedItem,
+    },
```

Comment:
What about creating a factory in sdk so you don't need to specify the created date every time?

Reply:
YES, sure!